tester.cc

```diff
@@ -3,7 +3,10 @@
 
 #include "uint.h"
 #include "sint.h"
-
+#include <iostream>
+#include <fstream>
```

Comment:
I don't believe firrtl-sig needs to deal with files (including following lines).